docs/server/node/upgrade_guide/v5ToV6UpgradeGuide.mdx

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
this page has no sidebar

Reply:
Should we have a sidebar for this page? I deliberately not have a sidebar for this page, it's only linked within the page